examples/argocd-route.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     example: route
 spec:
+  applicationSet: {}
```

Comment:
@iam-veeramalla is this intentionally added here to the example route?

Reply:
ah no, good catch. I will get rid of it.